LondonBritishLibrary/orient/BLorient580.xml

```diff
@@ -31,21 +31,63 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </altIdentifier>
                         <altIdentifier>
                             <idno>Wright 194</idno>
-                            
+
                         </altIdentifier>
-                        
+
                     </msIdentifier>
-                    <msContents>
-                        <summary/>
-                        
-                        
-                    </msContents>
-                    
-                    
-                    
-                    
-                    
-                    
+<msContents>
+<msItem xml:id="ms_i1">
+<locus from="1r"/>
+<title type="complete" ref="LIT1850Malkea"/>
+<textLang xml:lang="gez"></textLang>
+
+<msItem xml:id="ms_i1.1">
+<locus from="1r"/>
+<title xml:lang="en" type="complete">Hymn to the blessed Virgin Mary</title>
+<incipit xml:lang="gez">ሰላም፡ ሰላም፡ ለዝክረ፡ ስምኪ፡ ሐዋዝ።</incipit>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.2">
+<locus from="13r"/>
+<title xml:lang="en" type="complete">To our Lord</title>
+<incipit xml:lang="gez">ሰላም፡ ለዝክረ፡ ስምከ፡ ስመ፡ መሐላ፡ ዘኢይኄሱ።</incipit>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.3">
+<locus from="23r"/>
+<title xml:lang="en" type="complete">To our Lord</title>
+<incipit xml:lang="gez">ሰላም፡ ለዝክረ፡ ስምከ፡ ዘኢረከቡ፡ ተፍጻሜተ፡ መላእክተ፡ ሰማይ፡ ወምድር፡ እለ፡ ለመዱ፡ ስብሐተ።</incipit>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.4">
+<locus from="30v"/>
+<title xml:lang="en" type="complete">To St Michael</title>
+<incipit xml:lang="gez">ሰላም፡ ዝክረ፡ ስምከ፡ ምስ፡ ስመ፡ ልዑል፡ ዘተሳተፈ።</incipit>
+<textLang xml:lang="gez"></textLang>
```

Comment:
and ዝክረ፡

Reply:
Done

Reply:
I think that it would be better to encode the words within the opening and the closing tags, otherwise, it is not completely clear what is referred here.

Reply:
OK, I didn't know it is possible.